src/main/resources/application.properties

```diff
@@ -11,4 +11,6 @@ logging.pattern.console=%d{yyyy-MM-dd HH:mm:ss.SSS} [%thread] %-5level %logger{3
 
 management.endpoints.web.exposure.include=health, info, metrics
 management.endpoints.web.base-path=/
-management.endpoints.web.path-mapping.health=healthw
\ No newline at end of file
+management.endpoints.web.path-mapping.health=healthw
+
+server.error.include-stacktrace=never
```

Comment:
Is there a way we can include the server stack trace if we pass a log level of trace or really low level? It seems that this could actually be quite useful when debugging certain scenarios.

Reply:
This should be off for production otherwise it dumps the message as we saw a couple of weeks back.  I'm not sure how or when it got removed so I'm just putting it back in place. I'm not sure about changing it in production at run time, but that is a separate discussion for a separate issue.